api/v1alpha1/oadp_types.go

```diff
@@ -257,7 +257,7 @@ type VolumeOptions struct {
 	//+optional
 	CacheAccessMode string `json:"cacheAccessMode,omitempty"`
 	// moverSecurityContext allows enabling the PodSecurityContext used in
-	// the application pod(s)
+	// the application pod(s); default value is true
 	//+optional
 	MoverSecurityContext *bool `json:"moverSecurityContext,omitempty"`
```

Comment:
@eemcmullan Are you setting it to true by default in DPA ?

Reply:
@shubham-pampattiwar I avoided setting in DPA since I know we've discussed about other issues not changing DPA ourselves. We only need this value in the configmap for VSM controller to read, so I thought this way would be the most simple to get that across. If this field is not nil, then the user has set it, so default won't be used.

Reply:
Not against current behavior.. just we haven't had a discussion on hiding defaults in awhile.. and BackupImages field have similar behavior. nil = true.

Reply:
I guess having CRD markers do the default filling would also count as "changing dpa" from the yaml used... marking as resolved per prior discussions.